### ISSUE TITLE
Your Riot is misconfigured Invalid homeserver discovery response

### ISSUE THREAD

**Issue by student-sx** [vector-im/riot-web#13036]
### Description

When I start riot-web I get the following error message:

![Error](https://user-images.githubusercontent.com/18629143/78510079-fb1af580-7792-11ea-917a-fbcf4cda0451.png)

### Steps to reproduce

Install riot on ubuntu 19.04 following the instructions on https://riot.im/download/desktop/ 
(using xmonad as window manager).

Then start `riot-web` from command line and get the error message above.

**Comment by PipsqueakH** [comment]
same on manjaro, and the problem is it seems that there is nowhere to configure the homeserver?

**Comment by dbkr** [comment]
Check you can reach matrix.org: https://github.com/vector-im/riot-web/issues/11154 means that this is the error you get if the default homeserver is inaccessible.

**Comment by kortschak** [comment]
matrix.org appears to be unreachable right now (as does vector.im) (tinfoil hat time).

**Comment by 5p4r70n** [comment]
i can access matrix.org but my riot desktop clint shows //Your Riot is misconfigured
Invalid homeserver discovery response // this

**Comment by Krysme** [comment]
This is a serious problem for users in China, since the official server is blocked by the GFW, but my homeserver isn't. I was planning to set the server after the software has been launched, but the software is blocked at startup.

**Comment by turt2live** [comment]
This sounds like a duplicate of https://github.com/vector-im/riot-web/issues/11154
